requirements.txt

```diff
@@ -1,7 +1,7 @@
-PyPrint>=0.2.5,==0.2.*
-Pygments>=2.1.3
+PyPrint~=0.2.5
```

Comment:
Why do I get the feeling that we're jumping from syntax to syntax in a loop every 2-3 weeks ...